.gitignore

```diff
@@ -1,6 +1,6 @@
-# Extras
+# Env related
 .env
```

Comment:
Should `.env` still be ignored? Is it not a way to provide fallback values?

From what I'm thinking `.env` could have the place of `.env.example` (if .example is not being used for anything else) - having the explanation of the variables and a default configuration as well!

Reply:
Either `.env` or `.env.test` - don't know which is more appropriate.

Reply:
After taking a look at [the docs](https://www.npmjs.com/package/dotenv-flow) I believe that `.env` should not be ignored and added with the example values (preferably ones that also work, as much as possible, without exposing any sensitive data) - basically `.env` should be what `.env.example` is at the moment.